hack/generate-manifest-flow-aggregator.sh

```diff
@@ -20,24 +20,26 @@ function echoerr {
     >&2 echo "$@"
 }
 
-_usage="Usage: $0 [--mode (dev|release)] [-fc|--flow-collector <addr>] [-ch|--clickhouse] [--keep] [--help|-h]
-Generate a YAML manifest for the Flow Aggregator, using Kustomize, and print it to stdout.
+_usage="Usage: $0 [--mode (dev|release)] [-fc|--flow-collector <addr>] [-ch|--clickhouse] [--verbose-log] [--help|-h]
+Generate a YAML manifest for the Flow Aggregator, using Helm and Kustomize, and print it to stdout.
         --mode (dev|release)            Choose the configuration variant that you need (default is 'dev').
         --flow-collector, -fc <addr>    Specify the flowCollector address.
                                         It should be given in format IP:port:proto. Example: 192.168.1.100:4739:udp.
         --clickhouse, -ch               Enable exporting flow records to default ClickHouse service address.
-        --keep                          Debug flag which will preserve the generated kustomization.yml.
         --coverage                      Generate a manifest which supports measuring code coverage of the Flow Aggregator binaries.
         --verbose-log                   Generate a manifest with increased log-level (level 4) for the Flow Aggregator.
                                         This option will work only with 'dev' mode.
         --help, -h                      Print this message and exit.
 
 In 'release' mode, environment variables IMG_NAME and IMG_TAG must be set.
 
-This tool uses kustomize (https://github.com/kubernetes-sigs/kustomize) to generate manifests for
-running Antrea on Windows Nodes. You can set the KUSTOMIZE environment variable to the path of the
-kustomize binary you want us to use. Otherwise we will look for kustomize in your PATH and your
-GOPATH. If we cannot find kustomize there, we will try to install it."
+In 'dev' mode, environment variable IMG_NAME can be set to use a custom image.
+
+This tool uses Helm 3 (https://helm.sh/) and Kustomize (https://github.com/kubernetes-sigs/kustomize)
+to generate manifests for Antrea. You can set the HELM and KUSTOMIZE environment variable to
+the path of the helm and kustomize binary you want us to use. Otherwise we will download the
```

Comment:
s/environment variable to the path of the helm and kustomize binary you want us to use/environment variables to the paths of the helm and kustomize binaries you want us to use

Reply:
Updated, thanks!